spec/lib/sms_api_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+require "savon/mock/spec_helper"
+
+describe SMSApi do
+  let(:api) { SMSApi.new }
+
+  context "#sms_deliver" do
+    include Savon::SpecHelper
+
+    before do
+      endpoint = "https://intranet.lorca.es/wssms/wssms.asmx?WSDL"
+      allow(Rails.application.secrets).to receive(:sms_end_point).and_return(endpoint)
+      savon.mock!
+    end
+
+    after do
+      savon.unmock!
+    end
+
+    it "returns true when web service response is success" do
+      success = File.read("spec/fixtures/files/sms_api/success.xml")
+      savon.expects(:enviar_sms).with(message: :any).returns(success)
+
+      response = api.sms_deliver("666777888", "AAAA")
+
+      expect(response).to be_truthy
```

Comment:
Is `be_truthy` the same as `be_present`? If so, I'd prefer the latter because in the code we use things like `response.present?` but not `response.truthy?` :thinking:.